src/api.js

```diff
@@ -49,9 +50,10 @@ class Api extends EventEmitter {
   consumePersistentEvent (eventName, consumerName, consumer, options) {
     const exchange = this.options.toExchangeName(eventName)
     const queue = this.options.toQueueName(eventName, consumerName)
+    const args = options && options.args
     return this.pool.acquire().then(channel =>
       channel.declareExchange(exchange, { type: 'fanout' })
-        .then(() => channel.declareQueue(queue))
+        .then(() => channel.declareQueue(queue, { args }))
```

Comment:
there could be case when args are undefined
in such case I think we mustn't pass `{ args }` as second arg to `declareQueue`